lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -51,6 +54,15 @@ export class RunsOverviewModel extends OverviewPageModel {
         this._eorReasonsFilterModel.observe(() => this._applyFilters());
         this._eorReasonsFilterModel.visualChange$.observe(() => this.notify());
 
+        this._runDurationFilterModel = new DurationInputModel();
+        this._runDurationFilterModel.observe(() => this._applyFilters());
+        this._runDurationOperatorSelectionModel = new ComparisonOperatorSelectionModel();
+        this._runDurationOperatorSelectionModel.observe(() => {
+            if (this._runDurationFilterModel.value !== null) {
```

Comment:
What if I want to clear the value of the filter? Only this one?
I think you'll have to do the same as I did for time range filter

Reply:
refactored